content/brand/typography.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Typography
+permalink: /brand/typography/
+layout: layouts/page
+sidenav: false
+tags: brand
+eleventyNavigation:
+  parent: brand
+  key: typography
+  order: 4
+  title: Typography
+
+---
+
+The primary font is **Helvetica Neue**, in bold and regular weights.
+We use one typeface to keep things simple, and use different font sizes and weights to create hierarchy.
```

Comment:
Awkward line break due to difference in markdown rendering between 11ty and Jekyll

Reply:
Fixed.